src/zrc/datamodel/models.py

```diff
@@ -508,3 +541,17 @@ def save(self, *args, **kwargs):
                 gen_id = self.__class__.objects.filter(identificatie=identificatie).exists()
             self.identificatie = identificatie
         super().save(*args, **kwargs)
+
+    def unique_representation(self):
+        return f'{self.identificatie}'
+
+
+def request_object_attribute(url: str, attribute: str, resource: Union[str, None] = None) -> str:
+    Client = import_string(settings.ZDS_CLIENT_CLASS)
+    client = Client.from_url(url)
+    client.auth = APICredential.get_auth(url)
+    try:
+        result = client.retrieve(resource, url=url)[attribute]
+    except (ClientError, KeyError):
```

Comment:
ah cool, that actually solves a couple of problems with non-existant keys already.

I'd like to see a `logger.warning(...)` statement to make this not-silent, though :)